SteakBot.Core/Extensions/CommandInfoExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Discord.Commands;
+
+namespace SteakBot.Core.Extensions
+{
+    public static class CommandInfoExtensions
+    {
+        public static string ToString2(this CommandInfo commandInfo)
```

Comment:
Since the most optimal solution is a custom method you might as well name it something more appropriate.
e.g. CustomToString, FormatResponse, CommandInfoToString, etc.

Reply:
Renamed to `CustomToString`.